src/ValidationError.js

```diff
@@ -2,7 +2,13 @@
 let strReg = /\$\{\s*(\w+)\s*\}/g;
 
 let replace = str =>
-  params => str.replace(strReg, (_, key) => params[key] || '')
+  params => str.replace(strReg, (_, key) => {
+    const val = params[key]
+    if (typeof val === 'number') {
+      return val
+    }
+    return val || ''
```

Comment:
Though one concern doing `String(val)` instead would be breaking backwards compatibility. This would be a big visibility change...especially if users are passing in undefined values unintentionally and still receiving decent looking results. Hopefully that makes sense.